### ISSUE TITLE
Consider moving field help text up next to label

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#948]
## Problem
Sometimes our help text is pretty far removed from the label and input. (e.g. long text).

## Proposed solution
https://medium.com/@sarahcodes/move-field-description-below-label-in-drupal-8-6c2c607164f6

**Comment by briand44** [comment]
This came up in the ITS meeting today in reference to the alert category field. The help text appears at the bottom of the list of radio buttons and they would prefer it to show under the label.

**Comment by GaryRidgway** [comment]
Was doing some work for its and [this seemed to work for me](https://github.com/uiowa/uiowa/pull/7219/commits/1c966b44274b5a47b18a63e4fefbae55356d34c6#diff-6aca303b33f863a6f3648ed30b3628e00bcf7256c3a57e357d6a126e4cddd874R645-R653). I think in order to do this for all other fields and fieldsets we would just need to know what fields or fieldsets we would or wouldn't want this to occur on.

**Comment by joewhitsitt** [comment]
So @GaryRidgway your commit for the ITS work is going to move the help text for all radios across the platform. Just confirming that was intentional https://github.com/uiowa/uiowa/commit/1c966b44274b5a47b18a63e4fefbae55356d34c6

**Comment by GaryRidgway** [comment]
Yes it was intentional, I thought that made more sense. Would you like to veto?

**Comment by joewhitsitt** [comment]
I wouldn't mind a bit more scope creep actually if we aren't making this specific to ITS. I'd like to see this for long text/long text with summary fields too.

**Comment by bspeare** [comment]
@GaryRidgway This appears to affect webform positioning options, so I would like to see this scoped further to the node edit screen or exactly what you are targeting.

**Comment by GaryRidgway** [comment]
So the exact and only things we want this on right now are:
-radios
-text fields with summary
the scope creep to me is really deciding the exclusions or inclusions. Do we want only node edit? layout builder? fragments?

I think my paths forward here are either more specificity, or just do the work for this issue in another PR

**Comment by joewhitsitt** [comment]
Based on EoD and scoping this so it doesn't affect webforms, I remembered that we have done some scoping before with https://github.com/uiowa/uiowa/blob/main/docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module#L420
